core_backend/app/__init__.py

```diff
@@ -158,4 +167,14 @@ def create_app() -> FastAPI:
     metrics_app = create_metrics_app()
     app.mount("/metrics", metrics_app)
 
+    if not SENTRY_DSN or SENTRY_DSN == "" or SENTRY_DSN == "https://...":
+        logger.warn("No SENTRY_DSN provided. Sentry is disabled.")
```

Comment:
`logger.warn` is deprecated. Should we `logger.warning` ?

Reply:
Yes! Thanks for picking it up. I'll push this change and commit.